public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -90,6 +85,7 @@ export class EhrViewComponent implements OnInit, OnDestroy {
       // a empty search returns top ordered by count_value desc
       // Note, we save this in its own subscription so we can unsubscribe when they start typing
       // and these results don't trump the search results in case they come back slower
+      this.totalParticipants = this.ehrDomain.participantCount;
```

Comment:
The participant count needs to be ehr and domain specific and just not the total participant count across.